x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/HttpCertificateCommand.java

```diff
@@ -979,6 +1039,31 @@ private static Integer readKeySize(Terminal terminal, int keySize) {
         });
     }
 
+    private static List<String> readKeyUsage(Terminal terminal, List<String> defaultKeyUsage) {
```

Comment:
The current implementation of `readKeyUsage` will not allow users to remove the `keyUsage`, but rather only to change the default value. I think this is okay, otherwise it would defeat the purpose of adding it in the first place (but I may oversee a use case where this could be a requirement).

Let me know if you think we should allow it to be overridden to a "none".

Reply:
I think for `http` it's fine - I'm not even sure we _need_ to let them change it.
If they want unlimited options they can use `openssl` (and suffer in pain)